### PR TITLE
tp: expose flamegraphs through PerfettoSQL

### PR DESCRIPTION
**Pull request by LalitMaganti**

### CODE REVIEW

File: test/trace_processor/diff_tests/syntax/flamegraph_tests.py [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+                   '' AS g
+            WHERE FALSE
+          ));
+          SELECT name, depth, g
+          FROM fg(__intrinsic_flamegraph_config(
+            'value', 'value', 'view', 'TOP_DOWN', 'grouping', 'g'));
+        """,
+        out=Csv("""
+        "name","depth","g"
+      """))
```
Comment (safayat-google) [inline test/trace_processor/diff_tests/syntax/flamegraph_tests.py:402]: nit, add at least one test to cover negative values/cumulative values?

**Review by safayat-google — APPROVED** [review]
